src/scale.ts

```diff
@@ -264,11 +264,14 @@ export function isExtendedScheme(scheme: string | ExtendedScheme): scheme is Ext
 }
 
 export interface Scale {
+  /**
+   * The type of scale. <br/> •  For a _quantitative_ field, supported quantitative scale types  are `"linear"` (default), `"log"`, `"pow"`, `"sqrt"`, `"quantile"`, `"quantize"`, and `"threshold"`. <br/> • For a _temporal_ field without `timeUnit`, the scale type should be `"time"` (default) or `"ordinal"`. <br/>  • For _ordinal_ and _nominal_ fields, the type is always `"ordinal"`. <br/>Unsupported values will be ignored.
```

Comment:
Since this gets markdownify, I think we should get rid of `<br/>` and manual bullet listing and instead make something like

```
The type of scale.

 - For a _quantitative_ field, supported quantitative scale types  are `"linear"` (default), `"log"`, `"pow"`, `"sqrt"`, `"quantile"`, `"quantize"`, and `"threshold"
- For a _temporal_ field without `timeUnit`, the scale type should be `"time"` (default) or `"ordinal"`.
...
```

and check if the output still looks reasonable.